### PR TITLE
Unpackage the examples

### PR DESCRIPTION
**Pull request by MayhemYDG**
Bonus:
- I've removed `@babel/polyfill`, it was only used for the tests, and we don't need it.
- `npm run dev-server` -> `npm start`
- Had to change the `name` of the root package.json file, otherwise `npm i -D .\packages\react-data-grid-addons\` would complain about two package.json files having the same `name` field.

### CODE REVIEW

**Comment by amanmahajan7** [comment]
Needs catchup

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-data-grid",
+  "name": "rdg",
```
[the remark this answers is not in the document]
Comment (MayhemYDG) [reply package.json]: Changed it back, but I can't guarantee that it'll always work.

File: package.json [hunk 2]
```diff
@@ -33,19 +33,25 @@
     "eslint-plugin-react": "^7.12.4",
     "eslint-plugin-react-hooks": "^1.5.1",
     "faker": "^4.1.0",
+    "immutability-helper": "^2.9.1",
```
[the remark this answers is not in the document]
Comment (MayhemYDG) [reply package.json]: image
We only import it.

**Comment by amanmahajan7** [comment]
What is the advantage of moving examples to the root directory? If we are using lerna then it makes sense to have a separate private repository. Are you considering adding storybook and use these examples for documentation also?

**Comment by MayhemYDG** [comment]
It makes it easier to manage, as previously when we updated rdg and rdg-addons via lerna, it didn't update rdg-examples with the new versions. We also don't need to keep track of its other dependencies, and we don't need to bootstrap it with lerna anymore. It's one less independent thing to worry about basically.

Right now the examples are just for dev convenience, so I don't have plans to turn them into a storybook or anything. We could strip it down to the essentials, not a high priority though.

**Pull request merged**